api/proto/index/v2/index.proto

```diff
@@ -0,0 +1,161 @@
+syntax = "proto3";
+
+import "google/protobuf/timestamp.proto";
+import "google/protobuf/empty.proto";
+
+package index.v2;
+option go_package = "./index/v2;index";
+
+// Service provides an external API for managing the index records
+service Service {
+  // Create allows to new records for a search.
+  rpc Create(CreateRecordsRequest) returns (CreateRecordsResult);
+  // Create allows to create a new records, streaming the records data in the request
```

Comment:
CreateWithStreamData allows to create new index records by streaming the records.

Reply:
yes